src/parse_clinvar_xml.py

```diff
@@ -162,7 +185,9 @@ def parse_clinvar_tree(handle, dest=sys.stdout, multi=None, verbose=True, genome
         ])
 
         list_significance= [
-            x.text.lower() for x in elem.findall('.//ClinVarAssertion/ClinicalSignificance/Description') if x is not None
+            pathogenicity_mapping[x.text.lower()] for x in
+            elem.findall('.//ClinVarAssertion/ClinicalSignificance/Description') if x is not None
+            and x.text.lower() in pathogenicity_mapping
```

Comment:
This might be a little easier to read as a normal loop rather than a comprehension, but I'm okay either way:

```python
list_significance = []

for x in elem.findall('.//ClinVarAssertion/ClinicalSignificance/Description'):
    if x is not None and x.text.lower() in pathogenicity_mapping:
        list_significance.append(x.text.lower())
```